docs/tutorials/loading_data.ipynb

```diff
@@ -13,8 +13,13 @@
    "metadata": {},
```

Comment:
<p>nit: remove "see the"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-dask/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>